subsys/storage/flash_map/flash_map_shell.c

```diff
@@ -23,18 +23,18 @@ static void fa_cb(const struct flash_area *fa, void *user_data)
 {
 	struct shell *shell = user_data;
 
-	shell_print(shell, "%-4d %-8d %-20s  0x%-10x 0x%-12x",
```

Comment:
This produces a warning. Please resolve it:

```
/home/nard/DBI-ICBIS-Economy-System/software/firmware/external/zephyr/subsys/storage/flash_map/flash_map_shell.c:26:21: warning: format '%x' expects argument of type 'unsigned int', but argument 8 has type 'off_t' {aka 'const long int'} [-Wformat=]
   26 |  shell_print(shell, "%-4d %-8d %-20s %-20s 0x%-10x 0x%-12x",
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   27 |       fa->fa_id, fa->fa_device_id, fa->fa_dev_name,
   28 |       fa->fa_label, fa->fa_off, fa->fa_size);
      |                     ~~~~~~~~~~
      |                       |
      |                       off_t {aka const long int}
/home/nard/DBI-ICBIS-Economy-System/software/firmware/external/zephyr/include/zephyr/shell/shell.h:1056:35: note: in definition of macro 'shell_print'
 1056 |  shell_fprintf(_sh, SHELL_NORMAL, _ft "\n", ##__VA_ARGS__)
      |                                   ^~~
/home/nard/DBI-ICBIS-Economy-System/software/firmware/external/zephyr/subsys/storage/flash_map/flash_map_shell.c:26:50: note: format string is defined here
   26 |  shell_print(shell, "%-4d %-8d %-20s %-20s 0x%-10x 0x%-12x",
      |                                              ~~~~^
      |                                                  |
      |                                                  unsigned int
      |                                              %-10lx
```

Reply:
Done